AeroBody/SolarArray/Knowledge_Retention.md

```diff
@@ -0,0 +1,7 @@
+Knowledge Retention
+---
+
+
+For more information on IV curves, click [here.](IV_Curves.pdf) 
+
+For more information on angle calculations, click [here.](Documents/Angle_Calculations.pdf)
```

Comment:
pdf link yields a 404 error. Move the pdf into the `_static` folder and update the link here.